docs/source/WritingTests.rst

```diff
@@ -540,7 +540,7 @@ passed but there were non-related unexpected situations described in warning
 log.
 
 You may log something into the test logs using the methods in
-:mod:`avocado.test.Test.log` class attributes. Consider the example::
+:mod:`avocado.Test.log` class attributes. Consider the example::
 
     class output_test(test.Test):
```

Comment:
Here is `class output_test(Test):` , if it is the only problem, just change and rebase it and we've done.